app/assets/stylesheets/base.scss

```
@@ -0,0 +1,233 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "variable";
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+@keyframes FadeIn {
+  from {
+    opacity: 1;
+  }
+
+  to {
+    opacity: 0.8;
+  }
+}
+
+@keyframes indent {
+  from {
+    margin-left: 0;
+  }
+
+  to {
+    margin-left: 10px;
+  }
+}
+
+@keyframes popupSlideIn {
+  from {
+    opacity: 0.7;
+    transform: translateX(300px);
+  }
+  to {
+    opacity: 1;
+    transform: translateX(0);
+  }
+}
+
+@keyframes sliderSlideIn {
+  from {
+    left: 0;
+  }
+  to {
+    left: -100%;
+  }
+}
+
+header {
+  background-color: $white;
+}
+
+.header__top {
+  background-color: $header-top-background;
+  color: $header-top-color;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 160px;
+  height: 38px;
+  font-size: 15px;
+}
+
+.i18n {
+  display: flex;
+}
+
+.header__top hr {
+  border-color: $border-color;
+  margin: 0 10px;
+  height: 16px;
+  -webkit-transform:rotate(90deg);
+}
+
+.header__top a {
+  text-decoration: none;
+  color: $header-top-color;
+}
+
+.header__top a:hover {
+  cursor: pointer;
+  color: $primary-color;
+}
+
+.validation {
+  display: flex;
+  position: relative;
+  align-items: center;
+}
+
+.validation__register,
+.validation__login {
+  text-decoration: none;
+  color: $header-top-color;
+}
+
+.header__center {
+  padding: 0 160px;
+  display: flex;
+  align-items: center;
+  margin: 26px 0;
+  justify-content: space-between;
+  position: sticky;
+}
+.logo-shop {
+  width: 40px;
+}
+.logo {
```

này cũng thế, cách ra huynh ơi
```suggestion
}

.logo {
```